code/modules/shuttles/escape_pods.dm

```diff
@@ -214,50 +214,67 @@ As such, a new tracker datum must be constructed to follow proper child inherita
 	//id_tag is the generic connection tag.
 	//TODO make sure you can't C4 this.
 
-	ex_act(severity)
-		return FALSE
+/obj/structure/machinery/embedded_controller/radio/simple_docking_controller/escape_pod/ex_act(severity)
+	return FALSE
+
+// TGUI stufferinos \\
+
+/obj/structure/machinery/embedded_controller/radio/simple_docking_controller/escape_pod/ui_interact(mob/user, ui_key, datum/nanoui/ui, force_open)
```

Comment:
Shouldn't this be handled via attack_hand?

Reply:
fixed